### PR TITLE
test and fix bug

### PR DESCRIPTION
**Pull request by dat26061994**
to day, i test demo and fix teamplate

### CODE REVIEW

File: resources/views/user/master.blade.php [hunk 1]
```diff
@@ -130,79 +130,12 @@
...
-                </div>
 
-            </div>
-        </div>
-    </div>
 
     <div class="footer-bottom">
         <div class="container">
             <div class="row">
                 <p class="pull-left">Copyright © 2013 E-Shopper. All rights reserved.</p>
```
Comment (4c0n) [inline resources/views/user/master.blade.php:138]: Is this a downloaded template?

File: public/user/css/main.css [hunk 2]
```diff
@@ -12,6 +12,11 @@ body {
   position: relative;
   font-weight:400px;
 }
+@media (max-width: 480px) {
```
Comment (4c0n) [inline public/user/css/main.css:15]: I think bootstrap provides a way to do this. http://getbootstrap.com/css/#responsive-utilities-classes
Angular can also help you with this: https://material.angularjs.org/latest/layout/introduction

But you can use media queries is you want.

File: app/Http/Controllers/UserController.php [hunk 3]
```diff
@@ -60,24 +61,36 @@ public function getEdit($id)
     {
         $userModel = new User();
         $user = $userModel->findUser($id);
-        return view('admin.user.edit', compact('user', 'id'));
+        $isset = count($user);
+        if ($isset > 0) {
+            return view('admin.user.edit', compact('user', 'id'));
+        } else {
```
Comment (4c0n) [inline app/Http/Controllers/UserController.php:67]: This else statement is empty.

File: app/Http/Controllers/UserController.php [hunk 4]
```diff
@@ -60,24 +61,36 @@ public function getEdit($id)
     {
         $userModel = new User();
         $user = $userModel->findUser($id);
-        return view('admin.user.edit', compact('user', 'id'));
+        $isset = count($user);
```
Comment (4c0n) [inline app/Http/Controllers/UserController.php:64]: A better name for this variable would be $count

**Review by 4c0n — CHANGES_REQUESTED** [review]

**Comment by dat26061994** [comment]
Thank you, i will fix it now

2017-04-26 17:51 GMT+07:00 Youri Westerman <notifications@github.com>:

**Pull request merged**